README.md

```diff
@@ -213,11 +213,11 @@ Example:
 

```

Comment:
- [ ] `README.md` file
```
## 📝 License <a name="license"></a>

This project is [MIT](./LICENSE) licensed.
```
I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**